202001.0/d9b92bea-6b3c-4f97-960c-e164237ce47f.md

```diff
@@ -96,42 +96,42 @@ Even if you have manual testers, don't forget the automated test cases. One day
 
-If you cannot do acceptance automation during development (as it is time consuming), at least take care of all things that will another person needs for writing autotests - write helpful methods (keywords) and data builders that generate test data and put them directly into the database. With this information, even a non-developer can write automated test scenarios provided that all the necessary tools, keywords, setup and clear-down methods are ready for use. Make sure that data builders are not only written small tests, create a proper data-builder that can, if the data is provided, fill in all the fields in your element (e.g. a product abstract) and fill mandatory fields by default if not set by the user. Think about the people using it after you. For you, during implementation you know and understand all the insides of the feature. Therefore, it will take an hour or two to prepare all such things. However, a developer who did not prepare it themself or a person, especially a non-developer may take several days just to understand what is needed and what affects changes have on existing test cases and so on. Certainly, developers can be asked to do it when needed it, but in this case it, will be unplanned time loss for a developer. Why add unnecessary distractions for the developers when proper preparation when time is less of an issues can prevent them.
+If you cannot do acceptance automation during development (as it is time consuming), at least take care of all things that another person will need for writing autotests - write helpful methods (keywords) and data builders that generate test data and put it directly into the database. With this information, even a non-developer can write automated test scenarios provided that all the necessary tools, keywords, setup and clear-down methods are ready for use. Make sure that data builders are not only written small tests, create a proper data-builder that can, if the data is provided, fill in all the fields in your element (e.g. a product abstract) and fill mandatory fields by default if not set by the user. Think about the people using it after you. For you, during implementation you know and understand all the insides of the feature. Therefore, it will take an hour or two to prepare all such things. However, a developer who did not prepare it himself or a person, especially a non-developer may take several days just to understand what is needed and what affects changes have on existing test cases and so on. Certainly, developers can be asked to do it when needed, but in this case it will be unplanned time loss for a developer. Why add unnecessary distractions for the developers when proper preparation when time is less of an issues can prevent them?
```

Comment:
I do not quite get what's going on in "Make sure that data builders are not only written small tests" sentence.